src/main/java/com/sfl/cronofy/api/model/request/FreeBusyRequest.java

```diff
@@ -144,8 +144,12 @@ public void setIncludeManaged(final Boolean includeManaged) {
         this.includeManaged = includeManaged;
     }
 
-    public List<String> getCalendarIds() {
-        return calendarIds;
+    public Object[] getCalendarIds() {
+        if (calendarIds == null) {
```

Comment:
I think it could be better if we can move the logic of this getter in to private utility method in `CronofyClientImpl`.
Even though you have written this because of encapsulation, but in java world, you better don't touch `POJO` classes...